CONTRIBUTING.md

```diff
@@ -1,11 +1,40 @@
-# How to Contribute
+# Contributing to Kubeflow Spark Operator
 
-We'd love to accept your patches and contributions to this project. There are
+Welcome to the Kubeflow Spark Operator project. We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
-## Code reviews
+## Follow the Code of Conduct
+
+Before any contribution you would like to make, please have a look at the [code of conduct](CODE_OF_CONDUCT.md), which details how contributors are expected to conduct themselves as part of the Kubeflow Spark Operator Community.
```

Comment:
Please can we refer the [Kubeflow website code of conduct](https://www.kubeflow.org/docs/about/contributing/#follow-the-code-of-conduct).
So we have 1 place where we should redirect contributors.
To avoid confusion, I don't think we need to create separate CODE_OF_CONDUCT.md file in Spark Operator repo.
WDYT @ChenYi015 @kubeflow/wg-data-leads @kubeflow/kubeflow-steering-committee ?

Reply:
Make sense, i will remove it.

Reply:
I had kept the `CODE_OF_CONDUCT.md` file and redirect it to the [Kubeflow website code of conduct](https://www.kubeflow.org/docs/about/contributing/#follow-the-code-of-conduct).